lib/forms-schema.js

```diff
@@ -99,6 +99,20 @@ const ConditionalPredicatesItemsSchema = (
       then: Joi.boolean().required(),
       else: Joi.allow(null),
     }),
+    min: Joi.label(
+      `${subject} - Conditionally ${action} Predicate - Between-Min`,
+    ).when('type', {
+      is: Joi.only('BETWEEN'),
+      then: Joi.number().required(),
+      else: Joi.allow(null),
```

Comment:
```suggestion
      else: Joi.any().strip(),
```

Better to strip this out if it's not required, that way it will get removed if the user changes the type after saving a BETWEEN rule.